packages/engine/lib/execution/src/runner/python/setup.sh

```diff
@@ -47,7 +47,28 @@ echo "Running $(python --version) from $(which python)"
 
 echo "Installing python dependencies"
 python -m pip install --upgrade pip
-python -m pip install -r "${SCRIPT_DIR}/requirements.txt"
+if uname -s | grep -q Darwin && uname -p | grep -q arm ; then
+  if ! brew --prefix openblas ; then
```

Comment:
I agree that brew is probably the way people should install it, but do we have to require that they do it through brew?

I see we're setting the `PKG_CONFIG_PATH` manually below so maybe for now it's fine to just assume people will use brew and wait until someone encounters a problem and then adjust with new info.. but still not 100% happy with it

Reply:
(general comment, we don't have to block merging on this)